File: csharp/src/Structural/Adapter/LooseLeafTea.cs
//LooseLeafTea.java - the adaptee

namespace Structural.Adapter {


public class LooseLeafTea {  
   public bool teaIsSteeped; 
    
   public LooseLeafTea() {
       teaIsSteeped = false;
   }
   
   public void steepTea() {
       teaIsSteeped = true;
   }
}

}

File: scala/src/main/scala/Structural/Adapter/LooseLeafTea.scala
package Structural.Adapter;


//LooseLeafTea - the adaptee

class LooseLeafTea {  
   boolean teaIsSteeped; 
    
   LooseLeafTea() {
       teaIsSteeped = false;
   }
   
   void steepTea() {
       teaIsSteeped = true;
       System.out.println("tea is steeping");
   }
}
